README.md

```diff
@@ -1,5 +1,10 @@
 # laminas-oauth
 
+> [!CAUTION]
```

Comment:
The readme file needs some more updates:

https://github.com/laminas/laminas-oauth/blob/5182456ec570c6dd6c04003349ab65d9bd553850/README.md?plain=1#L19-L25

https://github.com/laminas/laminas-oauth/blob/5182456ec570c6dd6c04003349ab65d9bd553850/README.md?plain=1#L27

And was there a discussion about whether something should be placed above the statement on the war or only below it?

Reply:
I don't remember any discussion about the order - most other archived packages have it before the Ukraine message

Reply:
The next problem is the documentation: What should happen with it? Delete it, add a info or what should happen? 
That is nonsense:

- https://docs.laminas.dev/laminas-crypt/
- https://docs.laminas.dev/components/#security

The questions must be answered _before_ a repository is archived.

Reply:
There aren't any docs for this package, but I agree that we need to iron out what to do about documentation of archived libs.

Pehaps a new section for 'Archived/Abandoned', list them there. Is there a way that mkdocs could inject a notice for all abandoned libs?

Reply:
I think we should create a checklist for this. Otherwise the question will come back later or something will be forgotten. Can be added here: https://github.com/laminas/.github